docs/development-loadbalancer.md

```diff
@@ -0,0 +1,157 @@
+# Azure LoadBalancer
```

Comment:
Can you document how the "Standard LB" support works? (Does it work with VMSS > 100, does it work with multiple VMSS > 100, does it do the same rule reconcilliation?)

Reply:
Yes, Standard LB might have some special behavior. Basically this PR is to move the document from k8s repository only. 

@feiskyer  Do you have some comment on this? Suppose we could add the information in follow up PR.

Reply:
yep, we may add it in a following PR.

@colemickens Standard load balancer supports multiple vmss/vmas with nodes up to 1000. It reconcile NSGs and Routes same like basic LB, but there're also differences of course:

- (1) It adds all nodes and masters (controlled by setting `excludeMasterFromStandardLB`) in same backend pools
- (2) It creates standard SKU floating IP
- (3) Because of (1), it doesn't support `service.beta.kubernetes.io/azure-load-balancer-mode`.